backend/common/src/main/java/ai/verta/modeldb/common/reconcilers/Reconciler.java

```diff
@@ -142,4 +144,8 @@ public void insert(T element) {
   public abstract void resync();
 
   protected abstract ReconcileResult reconcile(Set<T> objs);
+
+  public boolean isElementsNotDeleted() {
+    return !processingIdSet.isEmpty();
+  }
```

Comment:
```suggestion
  public boolean isEmpty() {
    return processingIdSet.isEmpty();
  }
```
It's better to use nomenclature common in the libraries around. It's much clearer than trying to think about what a given name means.

Reply:
FIxed